torch/csrc/distributed/rpc/init.cpp

```diff
@@ -225,6 +228,14 @@ PyObject* rpc_init(PyObject* /* unused */) {
                     return PyRRef::unpickle(t);
```

Comment:
Shouldn't we have an TORCH_CHECK here as well?

Reply:
If rref can not be pickled, it can not be unpickled as well? but I think it is not harm to add one more check, will add it